sim-lib/src/lib.rs

```diff
@@ -352,6 +358,16 @@ struct Payment {
     dispatch_time: SystemTime,
 }
 
+impl Payment {
+    fn activity_prefix(&self) -> String {
```

Comment:
nit: add a comment explaining usage here / that this should be added before a log with no space